cdapython/utils/utility.py

```diff
@@ -140,7 +146,9 @@ def unique_terms(
 
     if async_req is None:
         async_req = False
-    col_name = col_name
+
+    if limit is not None:
```

Comment:
what if the limit is still a very large number where we would still want to pull the results in pages?

Reply:
This right now is here as a hold over for users that use limit instead of page_size but i don't know if i should autopage for people over 100